DEVELOP.rst

```diff
@@ -29,11 +35,105 @@ set up your remote interpreter and test environment.
 Running the Tests
 =================
 
-You can run the tests like so::
+You can run the tests like::
+
+    vagrant ssh
+    cd /vagrant
+
+    # Run test suite
+    composer run test
+
+    # Run code style checks
+    composer run style
+
+    # Run code style checks
+    $ ./vendor/bin/phpcs ./src
+
+
+
+************
+Using Docker
+************
+
+Installation
+============
+
+Clone the project::
+
+    git clone git@github.com:crate/crate-dbal.git
+
+Setup environment::
+
+    composer install --prefer-source
+
+
+Running the Tests
+=================
+
+::
+
+    # Run CrateDB
+    docker run -it --rm --publish 4200:4200 --publish 5432:5432 crate/crate:nightly
+
+    # Run test suite
+    composer run test
+
+    # Run code style checks
+    composer run style
+
+    # Run code style checks
+    ./vendor/bin/phpcs ./src
+
+
+****************************
+Working on the documentation
+****************************
+
+- The documentation is written using `Sphinx`_ and `ReStructuredText`_.
+- Python>=3.7 is required.
+
+Change into the ``docs`` directory:
+
+.. code-block:: console
+
+    $ cd docs
+
+For help, run:
+
+.. code-block:: console
+
+    $ make
+
+    Crate Docs Build
+
+    Run `make <TARGET>`, where <TARGET> is one of:
+
+      dev     Run a Sphinx development server that builds and lints the
+              documentation as you edit the source files
+
+      html    Build the static HTML output
+
+      check   Build, test, and lint the documentation
+
+      qa      Generate QA telemetry
+
+      reset   Reset the build
+
+You must install `fswatch`_ to use the ``dev`` target.
+
+
+Continuous integration and deployment
+=====================================
+
+CI is `configured`_ to run ``make check`` from the ``docs`` directory.
+Please do not merge pull requests until the tests pass.
```

Comment:
I'd really be in favor of setting this up as a strict rule enforced by the CI and not a loose advise. Isn't it already enforced?

Reply:
this has been set up as a strict rule on all repositories. this advice is technically redundant now. I would be okay with removing the second sentence

we should remove it from all docs projects while we're at it

what do you think, @amotl?

Reply:
Sure thing. I've addressed your suggestion and also propagated that change to https://github.com/crate/crate-docs/pull/62. I believe there is no need to explicitly step through all repositories now, though.